packages/cli/lib/run/browsers/chrome.js

```diff
@@ -28,6 +28,7 @@ export default class ChromeBrowser extends BaseBrowser {
   get arguments() {
     return [
       `--user-data-dir=${this.tmpdir}`,
+      '--headless',
```

Comment:
Was going to say we should get rid of this now but I don't think it matters since this cli is going away.